### DISCUSSION TITLE
Actions page on iOS app isn’t displaying properly

### DISCUSSION THREAD

**Discussion by bitcoin-tools** [community/community#121417] [Mobile] [answered]
Viewing the actions page of [this repo](https://github.com/bitcoin-tools/nodebuilder) looks weird.

![image](https://github.com/community/community/assets/156422466/541feebf-1041-4e85-aa65-9cd57a895f84)

Update - just closed and re-opened the app, and there's a different weird thing going on. The first row of the first entry is rendering above the top of the display area. (See the next 2 screenshots.)

![image](https://github.com/community/community/assets/156422466/70778ac3-ec8c-4747-8899-03cab3663479)

![image](https://github.com/community/community/assets/156422466/45c0a632-06b2-467f-b85d-a7907f243d04)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.159.0 (129045654)
Device: iPhone 15 Pro (iOS 17.4.1)
</details>

**Comment by Iron-Ham** [comment, marked as answer]
Hi @bitcoin-tools,

Thanks for reporting this bug! Reports like this one keep our app at a high level of quality. I'll open an internal ticket to track this and ensure that we tackle it soon. 


Thanks,
Hesham

**Reply by bitcoin-tools** [reply]
Thanks for ticketing it @Iron-Ham.

**Reply by Iron-Ham** [reply]
@bitcoin-tools a fix has been merged and should be shipping in this week's beta build. If you have access to the beta, keep your eyes out for it this week. Otherwise, it should be in production next week.
